pkg/apis/contrail/v1alpha1/kubemanager_types.go

```diff
@@ -82,6 +83,8 @@ type KubemanagerConfiguration struct {
 	RabbitmqUser          string       `json:"rabbitmqUser,omitempty"`
 	RabbitmqPassword      string       `json:"rabbitmqPassword,omitempty"`
 	RabbitmqVhost         string       `json:"rabbitmqVhost,omitempty"`
+	KeystoneSecretName    string       `json:"keystoneSecretName,omitempty"`
+	KeystoneInstance      string       `json:"keystoneInstance,omitempty"`
```

Comment:
There shouldn't be direct dependency on keystone CR in kubemanager controller and crd

Reply:
Keystone endpoint address should be a part of KubemanagerNodesConfiguration structure, like config, zookeeper ....

Reply:
+1